src/main/java/com/github/splendor_mobile_game/game/model/Game.java

```diff
@@ -78,17 +85,50 @@ public boolean startGame(Room room) {
         this.onyxTokens     = createTokenList(TokenType.ONYX);
         this.goldTokens     = createTokenList(TokenType.GOLD_JOKER);
 
+        //If startGame will be called second time, we need to clear decks
+        //Maybe will be good idea to make startGame private and call it in constructior?
```

Comment:
I think that's a good idea. Right now we are calling in the constructor methods to populate arrays and then upon calling `startGame` they're cleared and setup again.

Reply:
You can't call startGame() second time. It can be called only once, only if user clicks a button StartGame(). `It can be moved to constructor in future`. When game ends, `Game` object is deleted since only Room exists. Game is over so we don't need this object anymore.